components/ui/tabcounter/src/main/java/mozilla/components/ui/tabcounter/TabCounter.kt

```diff
@@ -19,8 +19,7 @@ import mozilla.components.support.ktx.android.view.onNextGlobalLayout
 import mozilla.components.support.utils.DrawableUtils
 import java.text.NumberFormat
 
-@Suppress("LargeClass")
-open class TabCounter @JvmOverloads constructor(
```

Comment:
Is there a reason that you're removing `open`?

Reply:
Because it was pointed by Android studio and I'm unable to find any reason to be let it be as is.
Of course I'm amateur, i can add it back if you want.

Reply:
yes, I think it would be better to keep `open` if we're not sure the original intention.  Thanks